src/sdk.ts

```diff
@@ -365,9 +368,12 @@ export class DefaultRelayingServices implements RelayingServices {
                         callForwarder: smartWallet.address,
                         data: unsignedTx.data,
                         tokenContract: smartWallet.tokenAddress,
-                        tokenAmount: await this.web3Instance.utils.toWei(
-                            tokenAmount.toString()
-                        ),
+                        tokenAmount: [null, undefined].includes(tokenAmount)
+                            ? undefined
```

Comment:
@antomor this is valid now since the server will interpret it as sponsored, no? (trying to recall why we made this change)

Reply:
IIRC we made this change to avoid the error in case of `tokenAmount` undefined (i.e. `tokenAmount.toString()` would raise an error). A more synthetic approach would be `tokenAmount?.toString()` but the final result is the same.